css/index.css

```diff
@@ -245,9 +490,66 @@ footer {
     }
 
     /* footer section */
-    footer {
-        height: auto;
-        padding-top: 2.5vh;
-        font-size: 2rem;
-    }
-}
+           footer {
+               display: flex;
+               justify-content: center;
+               flex-wrap: wrap;
+               height: 50vh;
+               font-size: 2rem;
+               text-align: center;
+           }
+    
+           footer nav {
+               display: flex;
+               justify-content: center;
+               flex-wrap: wrap;
+               align-items: center;
+               width: 95%;
+           }
+    
+           footer nav .mailing-list {
+               width: 100%;
+               height: 13vh;
+           }
+    
+           footer nav .mailing-list h4 {
+               font-size: 2rem;
+               padding: 2.5%;
+           }
+    
+           footer nav .mailing-list input {
+               width: 100%;
+               font-size: 1.6rem;
+           }
+    
+           footer nav .mailing-list button {
+               width: 100%;
+               font-size: 1.6rem;
+           }
+    
+           footer nav .btm.btns {
+               width: 100%;
+               height: 20vh;
+               flex-wrap: wrap;
+           }
+    
+           footer nav .btm.btns a {
+               height: 5vh;
+               width: 100%;
+               display: flex;
+               justify-content: center;
+               font-size: 2.2rem;
+           }
+    
+           .copyright {
+               width: 75%;
+               height: 8vh;
+           }
+    
+           .copyright p {
+               font-size: 1.35rem;
+               display: flex;
+               justify-content: center;
+               align-items: center;
+           }
+}
```

Comment:
Overall awesome job! It is almost exactly what was given so definitely awesome job on getting this done!